src/Dropdown/Dropdown.svelte

```diff
@@ -145,7 +145,8 @@
     highlightedIndex = index;
   }
 
-  const dispatchSelect = () => {
+  const dispatchSelect = async () => {
+    await tick();
     dispatch("select", { selectedId, selectedItem });
   };
```

Comment:
Overall this works and you're welcome to ship it. Thanks for catching this miss by me.

Would you prefer to remove the `tick` and return the `selectedItem` directly from the `items` array? Although the `tick` is a negligibly short time, I fear it may cause a race condition somehow.

```suggestion
  const dispatchSelect = async () => {
    dispatch("select", {
      selectedId,
      selectedItem: items.find((item) => item.id === selectedId).text,
    });
  };
```

Reply:
Amended the commit to use this approach.